assets/src/edit-story/components/inspector/prepublish/components/checklistTab.js

```diff
@@ -243,81 +212,81 @@ const ChecklistTab = ({
     [renderRow]
   );
 
+  // Determine what to make visible based on checkpoint and present issues.
   const highPriorityLength =
     highPriority.length + (pages.lengths?.highPriority || 0);
+  const hasHighPriorityItems = Boolean(highPriorityLength);
+  const isHighPriorityVisible =
+    !isHighPriorityDisabledState && hasHighPriorityItems;
+
   const recommendedLength =
     recommended.length + (pages.lengths?.recommended || 0);
-  const hasHighPriorityItems = Boolean(highPriorityLength);
   const hasRecommendedItems =
-    Boolean(recommended.length) || Boolean(pages.lengths?.recommended);
+    Boolean(recommended.length) ||
+    Boolean(pages.lengths?.recommended) ||
+    (canOptimizeVideo && !areVideosAutoOptimized);
+  const isRecommendedVisible =
+    !isRecommendedDisabledState && hasRecommendedItems;
 
-  const isRecommendedDisabled =
-    isRecommendedDisabledState || !hasRecommendedItems;
-  const isHighPriorityDisabled =
-    isHighPriorityDisabledState || !hasHighPriorityItems;
+  const isEmptyView = useMemo(() => {
+    return (
+      currentCheckpoint === PPC_CHECKPOINT_STATE.UNAVAILABLE ||
+      isChecklistEmpty ||
+      (!isRecommendedVisible && !isHighPriorityVisible)
```

Comment:
There are the 3 different conditions that will show an empty state now that we don't have 'disabled' panels. 

1) unavailable checkpoint (0 state)
2) empty checklist checkpoint (cleared of issues) 
3) checkpoint hasn't yet reached 'all' condition but there's no more issues to resolve.